validation/static/styles.css

```diff
@@ -439,3 +439,8 @@ a:hover {
 .logout-link:hover, .login-link:hover {
   color: var(--custom-purple);
 }
+
+textarea {
```

Comment:
For the least amount of surprises in the future, you may want to make a class and apply it to the specific `<textarea>`(s) you want. That way, if you add `<textarea>`s in the future, you won't be asking --- why is this thinner than I think it should be!

Reply:
Absolutely, I completely forgot I could apply styles directly to text areas. Fixed!